oper8/cmd/run_health_check.py

```diff
@@ -0,0 +1,76 @@
+"""
+This is the main entrypoint command for running the operator
+"""
+# Standard
+from datetime import datetime, timedelta
+from pathlib import Path
+from typing import List, Optional, Type
+import argparse
+import importlib
+import os
+import signal
+
+# Third Party
+import yaml
+
+# First Party
+import alog
+
+# Local
+from .. import config, watch_manager
+from ..controller import Controller
+from ..deploy_manager import DryRunDeployManager
+from ..exceptions import ConfigError
+from ..watch_manager.python_watch_manager.threads.heartbeat import HeartbeatThread
+from .base import CmdBase
+
+log = alog.use_channel("MAIN")
+
+
+class RunHealthCheckCmd(CmdBase):
```

Comment:
hmmm, good point. I think renaming it to `CheckHeartbeatCmd` will be the cleanest/easiest solution instead of implementing the full watch manager aware health check. While I would like to have a full health-check command that can be implemented later.

I'll push a commit